script.js

```diff
@@ -0,0 +1,26 @@
+// Used Youtube and Chatgpt for help
+function fetchRandomDogImage() {
+  const url = "https://random.dog/woof.json";
+
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
+      const dogImageContainer = document.getElementById("dog-image-container");
+      dogImageContainer.innerHTML = ""; // Clear the container
+
+      // Create an img element and set its source to the random dog image URL
+      const imgElement = document.createElement("img");
+      imgElement.classList.add("dog-image");
+      imgElement.src = data.url;

```

Comment:
The idea was to use classes to create the sprite. Try creating a class called `DogImage` with some properties like `this.src` and settting the `url` there.